dms/migrations/13.0.1/pre-migration.py

```diff
@@ -0,0 +1,88 @@
+    ("muk_dms_storage", "dms_storage"),
+    ("muk_dms_file_tag_rel", "dms_file_tag_rel"),
+    ("muk_dms_directory_tag_rel", "dms_directory_tag_rel"),
+    ("muk_dms_directory_star_rel", "dms_directory_star_rel"),
+    ("muk_security_access_groups", "dms_access_group"),
+    # Security Mixin relation tables
+    ("muk_dms_directory_groups_rel", "dms_directory_groups_rel"),
+    ("muk_dms_directory_complete_groups_rel", "dms_directory_complete_groups_rel"),
+]
+
+_field_renames = [
+    ("dms.category", "dms_category", "parent_category", "parent_id"),
+    ("dms.category", "dms_category", "child_categories", "child_category_ids"),
+    ("dms.category", "dms_category", "tags", "tag_ids"),
+    ("dms.category", "dms_category", "directories", "directory_ids"),
+    ("dms.category", "dms_category", "files", "file_ids"),
+    ("dms.directory", "dms_directory", "root_storage", "root_storage_id"),
+    ("dms.directory", "dms_directory", "storage", "storage_id"),
+    ("dms.directory", "dms_directory", "parent_directory", "parent_id"),
+    ("dms.directory", "dms_directory", "company", "company_id"),
+    ("dms.directory", "dms_directory", "category", "category_id"),
+    ("dms.directory", "dms_directory", "child_directories", "child_directory_ids"),
+    ("dms.directory", "dms_directory", "tags", "tag_ids"),
+    ("dms.directory", "dms_directory", "user_stars", "user_star_ids"),
+    ("dms.directory", "dms_directory", "files", "file_ids"),
+    ("dms.directory", "dms_directory", "groups", "group_ids"),
+    ("dms.directory", "dms_directory", "complete_groups", "complete_group_ids"),
+    ("dms.tag", "dms_tag", "category", "category_id"),
+    ("dms.file", "dms_file", "directory", "directory_id"),
+    ("dms.file", "dms_file", "storage", "storage_id"),
+    ("dms.file", "dms_file", "company", "company_id"),
+    ("dms.file", "dms_file", "category", "category_id"),
+    ("dms.file", "dms_file", "tags", "tag_ids"),
+    ("dms.file", "dms_file", "groups", "group_ids"),
+    ("dms.file", "dms_file", "complete_groups", "complete_group_ids"),
+    ("dms.file", "dms_file", "mimetype", "res_mimetype"),
+    ("dms.storage", "dms_storage", "root_directories", "root_directory_ids"),
+    ("dms.storage", "dms_storage", "storage_directories", "storage_directory_ids"),
+    ("dms.storage", "dms_storage", "company", "company_id"),
+    ("dms.storage", "dms_storage", "storage_files", "storage_file_ids"),
+    ("dms.access.group", "dms_access_group", "directories", "directory_ids"),
+    ("dms.access.group", "dms_access_group", "parent_group", "parent_group_id"),
+    ("dms.access.group", "dms_access_group", "child_groups", "child_group_ids"),
+    ("dms.access.group", "dms_access_group", "groups", "group_ids"),
+    ("dms.access.group", "dms_access_group", "explicit_users", "explicit_user_ids"),
+    ("dms.access.group", "dms_access_group", "users", "user_ids"),
+]
+
+_field_add = [("complete_name", "dms.directory", "dms_directory", "char", False, "dms")]
```

Comment:
Why do you need to add this field this way?

Reply:
Because it was failing the migration due to missing `complete_name`. It was strange, but was the only solution I found in order to fix it.

If I remember properly is that complete_name is used in _order, and this was giving issues.

Reply:
OK, I see. Put a comment about that for avoiding the same question in the future.

Reply:
Done!